src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/tasklets/PopulationStatisticsLoaderStep.java

```diff
@@ -97,9 +97,6 @@
  *  }
  * }
  */
-@Component
-@StepScope
-@Import({JobOptions.class})
 public class PopulationStatisticsLoaderStep implements Tasklet {
```

Comment:
this class, PedLoaderStep and  PopulationStatisticsGeneratorStep have unused imports